docs/_docs/Starters Academy (LDSSA)/04-Bootcamp-Structure.md

```diff
@@ -1,18 +1,19 @@
 ---
-title: Bootcamp Structure and Rules
+title: Specialisation 1 Structure and Rules
```

Comment:
I think it is ok if we keep the USA version with `Specialization`. That's what we have everywhere, so it would be tough to change it all over.